content/topics/_index.en.md

```diff
@@ -0,0 +1,80 @@
+---
+title: "Topics"
+date: Thu Jun 10 11:16:31 EDT 2021
+description: "Key topics of interest in geodistributed systems."
+---
+
+*Note: This list of topics of interest is constantly changing, so check back frequently for updates! Suggested readings for each topic will also be expanded and modified.*
+
+## What is a Distributed System?
+- Consistency vs Availability
+- Data infrastructures of social media apps
+- Affordances and limitations of commercial cloud offerings (AWS, Google, etc)
+
+**Suggested Readings**
+
+Ghemawat, Gobioff & Leung (2003) [The Google File System](https://static.googleusercontent.com/media/research.google.com/en//archive/gfs-sosp2003.pdf)
+
+Corbett, et al (2012) [Spanner: Google’s Globally-Distributed Database](https://www.usenix.org/system/files/conference/osdi12/osdi12-final-16.pdf)
+
+Muralidhar et al. (2014) [f4: Facebook’s Warm BLOB Storage System](https://www.usenix.org/system/files/conference/osdi14/osdi14-paper-muralidhar.pdf)
```

Comment:
Other papers that might be useful: Bayou (anti-entropy), Scatter, Orleans (the Actor model) -- happy to provide refs.